mixins/_icons.scss

```diff
@@ -69,16 +73,19 @@
         content: "\64";
     }
     @if $symbol == 'down' {
-        content: "\76";
+        content: "\56";
     }
     @if $symbol == 'download' {
         content: "\2913";
     }
+    @if $symbol == 'expand' {
+        content: "\21f2";
+    }
     @if $symbol == 'eye' {
         content: "\69";
     }
     @if $symbol == 'football' {
-        content: "\1f3c8";
+        content: "\1f3c";
```

Comment:
Yeah, I don't know what happened here, this got cut off. We should probably not use emojis here.

TODO find if this is used anywhere and fix it.